modular_zubbers/modules/examine/examine_tgui.dm

```diff
@@ -0,0 +1,17 @@
+/datum/examine_panel/ui_data(mob/user)
+	var/list/data = ..()
+	var/datum/preferences/preferences = holder.client?.prefs
```

Comment:
You're pulling from client here- is FT not stored on the mob?

Reply:
NSFW flavor text isn't, flavor text is, I still have to fix that